test/evergreen.yml

```diff
@@ -2376,6 +2376,20 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: data-validation-test-checkpoint
```

Comment:
As there is an existing `checkpoint-test` task, can we rename the new task `checkpoint-data-validation-test` for ease of grouping on Evergreen UI?